testsuite/Rakefile

```diff
@@ -19,7 +19,19 @@ namespace :cucumber do
       profiles.split(',').each do |profile|
         include_profiles << "--profile #{profile} "
       end
-      cucumber_opts = %W[#{html_results} #{json_results} #{junit_results} -f rerun #{include_profiles} --out failed.txt -f pretty -r features]
+      tags = ENV['TAGS'] || ''
+      include_tags = ""
+      tag_list = tags.split(',')
+      tag_list.each_with_index do |tag, index|
+        if index == 0
+          include_tags << "--tags #{tag} or"
+        elsif index != tag_list.size - 1
+          include_tags << " #{tag} or"
+        else
+          include_tags << " #{tag}"
+        end
+      end
```

Comment:
If someone find a better way to construct "--tags @a or @b or @c" from a string "@a,@b,@c" please ilustrate me :)

Reply:
`'--tags ' + s.split(',').join(' or ')`, with `s = "@a,@b,@c"`

Reply:
Without the list: `'--tags ' + s.gsub(',', ' or ')`

Reply:
Awesome! I will change for this last version, thank you Tino! :)